app/package.json

```diff
@@ -42,44 +44,47 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "7.0.0",
-    "@angular/common": "7.0.0",
-    "@angular/compiler": "7.0.0",
-    "@angular/core": "7.0.0",
-    "@angular/forms": "7.0.0",
-    "@angular/http": "7.0.0",
-    "@angular/platform-browser": "7.0.0",
-    "@angular/platform-browser-dynamic": "7.0.0",
-    "@angular/router": "7.0.0",
-    "@biesbjerg/ngx-translate-po-http-loader": "^3.1.0",
+    "@angular/animations": "8.1.0",
+    "@angular/common": "8.1.0",
+    "@angular/compiler": "8.1.0",
+    "@angular/core": "8.1.0",
+    "@angular/forms": "8.1.0",
+    "@angular/http": "7.2.15",
```

Comment:
I believe the recommended HttpClient is now within the @angular/common/http, so we can remove this dependency